### ISSUE TITLE
v5: Unable to resolve imports when installed via NPM

### ISSUE THREAD

**Issue by kjy5** [dimsemenov/PhotoSwipe#1866]
I'm building a website in React using Vite. I installed PhotoSwipe 5.2.1 (official, not beta) from NPM this morning. However, when I tried to import the components and CSS files as shown [in the React tutorial](https://photoswipe.com/react-image-gallery/) like this:
``` javascript
import 'photoswipe/dist/photoswipe.css';
import PhotoSwipeLightbox from 'photoswipe/lightbox'
```
I get the following packaged unresolved error when I run `npm run build`
```
[vite:resolve] Missing "./dist/photoswipe.css" export in "photoswipe" package
```

I get the same error for the `photoswipe` module that's dynamically imported and the `lightbox` module too (the build just stops at the CSS file. 

How can I fix this? I have tried `npm ci` and uninstalling/re-installing with `npm i photoswipe --save`. Thanks!

**Comment by dimsemenov** [comment]
I'll investigate this tomorrow. package.json might need an adjustment https://github.com/dimsemenov/PhotoSwipe/blob/master/package.json Feel free to submit a pr if you see the solution.

**Comment by kjy5** [comment]
**Update:**
I looked into the `package.json` file and tried adding the CSS file to the exports so it now says
```json
"exports": {
    ".": "./dist/photoswipe.esm.js",
    "./lightbox": "./dist/photoswipe-lightbox.esm.js",
    "./dist/photoswipe.css": "./dist/photoswipe.css", // Added this line
  },
```
This did not help unfortunately. I then ran `vite --debug` to see what was going on and I see this
``` bash
  vite:resolve 0.71ms photoswipe/dist/photoswipe.css -> null +2ms
  vite:resolve 0.50ms photoswipe/lightbox -> null +1ms
  vite:resolve 0.17ms photoswipe -> null +0ms
```

I'm not very knowledgeable about how NPM modules are packaged. Hopefully, this will help you when debugging!

**Comment by dimsemenov** [comment]
Should be fixed by 5.2.2 

The change that you suggested does seem to work. Here is Vite demo - https://stackblitz.com/edit/vitejs-vite-e1vy3s?file=package.json 
Vite seems to ignore everything that's not in exports (if exports prop is present).

**Comment by kjy5** [comment]
I'm able to build PhotoSwipe now! Thank you!

I'm now experiencing a different problem where my images open in a new tab instead of the gallery, but that's separate from this problem. 

Thanks again!
